sessions/backend-read/products/next.config.js

```diff
@@ -0,0 +1,18 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  compiler: {
+    styledComponents: true,
+  },
+  reactStrictMode: true,
+  webpack(config) {
+    config.module.rules.push({
+      test: /\.svg$/i,
+      issuer: /\.[jt]sx?$/,
+      use: ["@svgr/webpack"],
+    });
+
+    return config;
+  },
```

Comment:
Yes, I think this would be an extra issue because it refers to all Next.js project in `web-exercises`. We thus need to change the [recipe for next](https://github.com/neuefische/web-exercises/tree/main/recipes/next/files) and update / migrate everything accordingly.

Reply:
I'll resolve this conversation for now.